lib/floe/workflow/states/choice.rb

```diff
@@ -37,17 +37,17 @@ def end?
 
         private
 
-        def validate_state!
-          validate_state_choices!
-          validate_state_default!
+        def validate_state!(workflow, payload)
+          validate_state_choices!(payload)
+          validate_state_default!(workflow, payload)
```

Comment:
Minor but isn't `@payload` set already here and we have an attr_reader?

I think you can drop this arg unless it is for another purpose

Reply:
changing.

But yea, I am not a fan of paylod or workflow.
(though it is pretty innocuous)